Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +230,18 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static bool containsInvalidCharacters(const String& str)
```

Comment:
I think you can call `String::contains()` which takes in a lambda

Reply:
That does look cleaner--thank you!